### PR TITLE
Rpc example

### PR DESCRIPTION
**Pull request by kjazgar**
Fixes | Closes | Resolves #

## Changes:
Added some example responses for implemented methods using sample parameters. 

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Testing
**Requires testing**

- [ ] Yes
- [x] No

**In case you checked yes, did you write tests??**

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by LukaszRozmej — APPROVED** [review]

File: src/Nethermind/Nethermind.JsonRpc/Modules/Eth/IEthRpcModule.cs [hunk 1]
```diff
@@ -47,41 +57,75 @@ public interface IEthRpcModule : IRpcModule
...
+            Description = "Returns accounts", 
+            IsSharable = true, 
+            ExampleResponse = "[\"0x9b96a7841d6e0b76872c85c86082959189a27342\"]")]
         ResultWrapper<IEnumerable<Address>> eth_accounts();
         
-        [JsonRpcMethod(IsImplemented = true, Description = "Returns current block number", IsSharable = true)]
+        [JsonRpcMethod(IsImplemented = true, 
+            Description = "Returns current block number", 
+            IsSharable = true, 
+            ExampleResponse = "0x0")]
```
Comment (MarekM25) [inline src/Nethermind/Nethermind.JsonRpc/Modules/Eth/IEthRpcModule.cs:81]: Could you return different examples than 0x0? :)

File: src/Nethermind/Nethermind.JsonRpc/Modules/Eth/IEthRpcModule.cs [hunk 2]
```diff
@@ -47,41 +57,75 @@ public interface IEthRpcModule : IRpcModule
...
-        
-        [JsonRpcMethod(IsImplemented = true, Description = "Returns account nonce (number of trnsactions from the account since genesis) at the given block number", IsSharable = true)]
-        Task<ResultWrapper<UInt256?>> eth_getTransactionCount(Address address, BlockParameter blockParameter = null);
-        
-        [JsonRpcMethod(IsImplemented = true, Description = "Returns number of transactions in the block block hash", IsSharable = true)]
-        ResultWrapper<UInt256?> eth_getBlockTransactionCountByHash(Keccak blockHash);
+        [JsonRpcMethod(IsImplemented = true, 
+            Description = "Returns account balance", 
+            IsSharable = true,
+            ExampleResponse = "0x0")]
```
Comment (MarekM25) [inline src/Nethermind/Nethermind.JsonRpc/Modules/Eth/IEthRpcModule.cs:87]: And here?

File: src/Nethermind/Nethermind.JsonRpc/Modules/Eth/IEthRpcModule.cs [hunk 3]
```diff
@@ -47,41 +57,75 @@ public interface IEthRpcModule : IRpcModule
...
         
         [JsonRpcMethod(IsImplemented = true, Description = "Returns number of transactions in the block by block number", IsSharable = true)]
         ResultWrapper<UInt256?> eth_getBlockTransactionCountByNumber(BlockParameter blockParameter);
         
-        [JsonRpcMethod(IsImplemented = true, Description = "Returns number of uncles in the block by block hash", IsSharable = true)]
-        ResultWrapper<UInt256?> eth_getUncleCountByBlockHash(Keccak blockHash);
+        [JsonRpcMethod(IsImplemented = true, 
+            Description = "Returns number of uncles in the block by block hash", 
+            IsSharable = true, 
+            ExampleResponse = "0x0")]
```
Comment (MarekM25) [inline src/Nethermind/Nethermind.JsonRpc/Modules/Eth/IEthRpcModule.cs:115]: And in more places :)

**Review by MarekM25 — CHANGES_REQUESTED** [review]
Thank you for this PR.
Please, ask @marcindsobczak how to generate docs from it. 
Please, add more sophisticated examples than just zeroes :)

**Review by marcindsobczak — APPROVED** [review]

**Review by MarekM25 — APPROVED** [review]

**Pull request merged**